source/install/WSL.rst

```diff
@@ -0,0 +1,25 @@
+Setting up Windows to use QIIME 2
+---------------------------------
+
+This section will demonstrate how to set up windows to allow QIIME 2 installation. Windows now allows users to install linux as a windows subsystem (WSL), the simplest way to use QIIME 2 on a machine with Windows OS is therefore to use this Windows subsystem for Linux.
+
+Please see the `official windows tutorial <https://docs.microsoft.com/en-us/windows/wsl/install-win10>`__ to install WSL2. Then install an appropriate linux distribution e.g. the latest stable version of Ubuntu. Once installation is complete run ``sudo apt-get update`` and ``sudo apt-get upgrade`` to ensure everything is up to date.
```

Comment:
I suggest removing the Ubuntu-specific instructions here, since Ubuntu is not a prerequisite.

```suggestion
Please see the `official Microsoft guide`_ to install the WSL. Once you have installed and set up the WSL, install a Linux distribution - we suggest the latest stable version of `Ubuntu`_.
```